docs/overview/glossary.md

```diff
@@ -53,28 +87,66 @@ Archway’s native IBC integration enables developers to future-proof their dApp
 <!-- J -->
 <!-- K -->
 <!-- L -->
+
+## Light Client Daemon (LCD)
+
+Light clients are software that allow users to interact with the blockchain by connecting to full nodes. These clients do not store the entire blockchain so they take significantly less storage and resources to run. 
+
+## Liquidity Mining 
+
+Liquidity Mining is when users lend their assets to a decentralized exchange to make it easier for other users to trade. In return these users receive an award usually in the form of a share of the fee or interest. 
 <!-- M -->
+
+## Minting 
+
+Minting is the process of creating new tokens on a blockchain through a process that includes validation, creation of new blocks and then recording that information to the blockchain. 
+
 <!-- N -->
 <!-- O -->
 
 <!-- P -->
 
+## Proof of Stake (PoS)
+
+Proof of Stake is a consensus mechanism that is used by Tendermint as well as other blockchains. In PoS, validators are selected by the proportion of the token they hold or their "stake" in the network. The validator to write the next block is chosen by random but a higher stake increases the chances that a specific validator will be chosen.  
+
 ## Pool account
 
 A common liquidity pool that Archway dApps can fund and then use to sponsor gas payments for end users.
 
 <!-- Q -->
 <!-- R -->
+
+## Remote Procedure Call (RPC)
+
+A RPC is a process that allows for the remote communication between servers and applications in different locations. This process allows for an application or dapp to interact and receive data about a blockchain via a server node.
```

Comment:
This should be "An RPC". https://www.grammarly.com/blog/indefinite-articles-a-and-an/

Reply:
I think 'A' would be correct here as it precedes a word with a consonant sound but to remove any confusion it might be clearer to remove the 'A' and start with "RPC is a process...".